spec/javascripts/components/intervention-spec.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-env jasmine, jquery */
+/* global GOVUK */
+
+describe('Intervention component', function () {
```

Comment:
Nitpick: I'd name this `describe` `An intervention component` or `The intervention component`.

Reply:
Makes sense. I thought that was the convention (looking at other components)